api/v1beta1/agentserviceconfig_types.go

```diff
@@ -102,6 +102,12 @@ type AgentServiceConfigSpec struct {
 	// +kubebuilder:validation:default:=disabled
 	// +optional
 	IPXEHTTPRoute string `json:"iPXEHTTPRoute,omitempty"`
+
+	// KubeconfigSecretRef is the reference to the Secret containing a kubeconfig of an external
+	// control plane (e.g. hypershift HostCluster).
```

Comment:
I think we should add some more words about what happens when you add this.

It's not clear that this changes where the controllers are looking to do reconcile.

Reply:
right, added some explanation.